src/components/app.js

```diff
@@ -144,18 +144,14 @@ class App extends Component {
     )
 
     if (
-      apiInfoData.locationLoaded &&
-      apiInfoData.contactLoaded &&
-      apiInfoData.socialLoaded &&
-      apiInfoData.footerLoaded &&
       apiInfoData.projectLoaded
     ) {
       return (
         <div styleName="blocks.container">
           <AppHeader
             handleClick={this.handleShow}
             onClick={this.handleHide}
-            title={apiInfoData.footerData.name}
+            title={apiInfoData.footerData?apiInfoData.footerData.name:null}
```

Comment:
`apiInfoData.footerData && apiInfoData.footerData.name`

No need to use ternary operator